nl.brixit.wiremapper.json

```diff
@@ -0,0 +1,59 @@
+{
+  "app-id": "nl.brixit.wiremapper",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "wiremapper",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--system-talk-name=org.bluez",
+    "--allow=bluetooth",
+    "--share=network"
+  ],
+  "modules": [
+    "python3-wiremapper.json",
+    {
```

Comment:
Leftover?

Reply:
no

Reply:
Well actually, I have no idea what it exactly does. But all examples I've found have this.

Reply:
I was talking about the empty module wiremapper that does nothing?

Reply:
yeah that was in all the examples I could find that included another json file, an empty block with only a name after it. I'll remove it.